### ISSUE TITLE
Generator function didn't preserve needed nodes, copying old replacements back in instead.

### ISSUE THREAD

**Issue by emelife** [tensorflow/tensorflow#34680]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
ubuntu 16.04 / 
python 3.6 /
tf 1.14.0 /
onnx 1.6.0 

**Describe the current behavior**
I want to convert siamfc network pb model to onnx model.
I build tensorflow siamfc project  from [https://github.com/bilylee/SiamFC-TensorFlow.git](url).
after a lot of step ,I got .ckpt.* file successfully, then I freeze to pb file.
this is a part of network below:
![image](https://user-images.githubusercontent.com/16054460/69796416-f9e82380-1208-11ea-8859-d74889fb0917.png)
then,I need to convert to onnx model.
by this cmd:
python -m tf2onnx.convert --input=./siamfc-edit.pb --inputs=inference/examplar_input:0,inference/instance_input:0 --outputs=inference/detection/add:0 --opset=11 --continue_on_error --fold_const --output=siamfc-edit.onnx --verbose
I think this cmd is correct, because I can convert a lot of other network successfully.
but when it convert , a lot of warning come up ,and never stop.
so that I can't convert to onnx .the warning is below 

......
2019-11-28 17:19:56.308150: W tensorflow/tools/graph_transforms/transform_utils.cc:448] Generator function didn't preserve needed nodes, copying old replacements back in instead.
2019-11-28 17:19:56.309946: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv2/b2/BatchNorm/beta/read/_10__cf__10 to be preserved.
2019-11-28 17:19:56.309960: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv2/b2/BatchNorm/gamma/read/_11__cf__11 to be preserved.
2019-11-28 17:19:56.309975: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv2/b2/BatchNorm/moving_mean/read/_12__cf__12 to be preserved.
2019-11-28 17:19:56.309983: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv2/b2/BatchNorm/moving_variance/read/_13__cf__13 to be preserved.
2019-11-28 17:19:56.310000: W tensorflow/tools/graph_transforms/transform_utils.cc:448] Generator function didn't preserve needed nodes, copying old replacements back in instead.
2019-11-28 17:19:56.319616: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv3/BatchNorm/beta/read/_15__cf__15 to be preserved.
2019-11-28 17:19:56.319670: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv3/BatchNorm/gamma/read/_16__cf__16 to be preserved.
2019-11-28 17:19:56.319678: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv3/BatchNorm/moving_mean/read/_17__cf__17 to be preserved.
2019-11-28 17:19:56.319686: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv3/BatchNorm/moving_variance/read/_18__cf__18 to be preserved.
2019-11-28 17:19:56.319718: W tensorflow/tools/graph_transforms/transform_utils.cc:448] Generator function didn't preserve needed nodes, copying old replacements back in instead.
2019-11-28 17:19:56.323429: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv4/b2/BatchNorm/beta/read/_25__cf__25 to be preserved.
2019-11-28 17:19:56.323456: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv4/b2/BatchNorm/gamma/read/_26__cf__26 to be preserved.
2019-11-28 17:19:56.323464: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv4/b2/BatchNorm/moving_mean/read/_27__cf__27 to be preserved.
2019-11-28 17:19:56.323471: W tensorflow/tools/graph_transforms/transform_utils.cc:441] Expected convolutional_alexnet/conv4/b2/BatchNorm/moving_variance/read/_28__cf__28 to be preserved.
2019-11-28 17:19:56.323479: W tensorflow/tools/graph_transforms/transform_utils.cc:448] Generator function didn't preserve needed nodes, copying old replacements back in instead.
......

so what's wrong with it or me?

I expect to know what's this warning mean,I found there are a lot of identity op as input op share for other op. 
![image](https://user-images.githubusercontent.com/16054460/69797447-ad9de300-120a-11ea-9dd0-cd01138d4fd2.png)
does it because of this?

I look forward to your reply . thanks a lot.

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
